includes/admin/class-wc-payments-admin.php

```diff
@@ -659,6 +660,13 @@ public function enqueue_payments_scripts() {
 				$this->get_js_settings()
 			);
 
+			$payment_method_definitions = rawurlencode( PaymentMethodUtils::get_payment_method_definitions_json() );
+			wp_add_inline_script(
+				'WCPAY_ADMIN_SETTINGS',
+				"const woopaymentsPaymentMethodDefinitions = JSON.parse( decodeURIComponent( '" . esc_js( $payment_method_definitions ) . "' ) );",
```

Comment:
I think we should stick to using `var` or `window.` to assign these global constants.
On one hand, for slightly better historic browser support (even tho that's a silly argument, [because almost every browser supports it](https://caniuse.com/const)), but also because `var` and `window.` allow overwrite of the variable without errors, in case it gets added more than once.

```suggestion
				"window.woopaymentsPaymentMethodDefinitions = JSON.parse( decodeURIComponent( '" . esc_js( $payment_method_definitions ) . "' ) );",
```

Reply:
Ahh...good point about the overwriting thing. I'll commit the suggestion.